release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -24,16 +24,9 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 You must use {op-system} machines for the control plane, and you can use either {op-system} or {op-system-base} for compute machines.
 //Removed the note per https://issues.redhat.com/browse/GRPA-3517
 
-For {product-title} 4.12 on `x86_64` architecture, Red Hat has added a 6-month Extended Update Support (EUS) phase that extends the total available lifecycle from 18 months to 24 months. For {product-title} 4.12 running on 64-bit ARM (`aarch64`), {ibm-power-name} (`ppc64le`), and {ibm-z-name} (`s390x`) architectures, the EUS lifecycle remains at 18 months.
+For {product-title} {product-version}, Red Hat offers a 6-month Extended Update Support (EUS) phase that extends the total available lifecycle from 18 months to 24 months on all supported architectures, including x86_64, 64-bit ARM (aarch64), IBM Power® (ppc64le), and IBM Z® (s390x) architectures. Beyond this, Red Hat also offers a 12-month additional EUS add-on, denoted as Additional EUS Term 2, that extends the total available lifecycle from 24 months to 36 months. The Additional EUS Term 2 is also available on all architecture variants of OpenShift Container Platform.
```

Comment:
```suggestion
For {product-title} {product-version}, Red Hat offers a 6-month Extended Update Support (EUS) phase that extends the total available lifecycle from 18 months to 24 months on all supported architectures, including `x86_64`, 64-bit ARM (`aarch64`), BM Power® (`ppc64le`), and IBM Z® (`s390x`) architectures. Beyond this, Red Hat also offers a 12-month additional EUS add-on, denoted as Additional EUS Term 2, that extends the total available lifecycle from 24 months to 36 months. The Additional EUS Term 2 is also available on all architecture variants of {product-title}.
```

Reply:
BM Power® ? Is there still supposed to be an I?

Reply:
Yeah, typo. We actually have attributes: {ibm-power-name} and {ibm-z-name}, which should be used as the IBM names recently went through a change.